prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+"""
+Get guitar info from user and display in meaningful manner.
+"""
+from guitar import Guitar
+
+
+def main():
+    guitars = []
+    name = input("Name:")
+
+    while name != "":
+        year = int(input("Year:"))
```

Comment:
I used try, except in these to prompt if invalid input. Double check if it is needed...